gui/src/components/Layout.svelte

```diff
@@ -1,27 +1,45 @@
 <script lang="ts">
-  import * as router from 'svelte-spa-router';
   import VersionInfo from './VersionInfo.svelte';
   import NavbarButton from './nav/NavbarButton.svelte';
-  import FeedbackSVG from './mono/feedback.svelte';
+  import FeedbackSVG from './mono/FeedbackSVG.svelte';
+  import PreferencesSVG from './mono/PreferencesSVG.svelte';
+  import { guiTheme, setGuiTheme } from '../lib/gui-theme';
+  import { onMount } from 'svelte';
+  import * as router from 'svelte-spa-router';
 
-  const goHome = () => {
-    router.push('#/');
-  };
+  onMount(() => {
+    setGuiTheme($guiTheme);
+  });
 </script>
 
 <main>
   <nav>
     <header>
-      <NavbarButton title="Home" on:click={goHome}>
-        <img src="/deref-rounded-icon.png" alt="Deref" height="24px" />
+      <NavbarButton
+        title="Home"
+        on:click={() => {
+          router.push('#/');
```

Comment:
Maybe `NavbarButton` should have a `href` property, which would clean up all these `on:click` handlers.